pulp_smash/utils.py

```diff
@@ -86,6 +86,23 @@ def os_is_f27(cfg, pulp_host=None):
     return response.returncode == 0
 
 
+def os_is_fips(cfg, pulp_host=None):
```

Comment:
It's impossible for a function to "be FIPS." It would be nicer to have a pair of methods called `fips_is_supported` and `fips_is_enabled`.